apps/style/netsim/style.scss

```diff
@@ -410,15 +406,18 @@ svg {
       }
 
       .watermark {
-        color: #d3d3d3;
+        color: $medium-gray;
       }
 
-      .bit_counter {
+      .bit-counter {
         text-align: right;
-        color: darkgray;
+        color: $dark-gray;
+        &.oversized {
```

Comment:
Rule declaration should be preceded by an empty line<br>Nesting should be no greater than 4, but was 5<br>Selector should have depth of applicability no greater than 2, but was 4